.changeset/young-cups-dream.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/search-input': minor
```

Comment:
Should this be major?

Reply:
The only thing making it a major change technically is the renaming of `sizeVariant`. Everything else is backwards compatible

Reply:
I'm inclined to say we move this to size and make it a major change. MAYBE worth looking into how widely this component is being used currently to know how heavy of a lift this would be as a breaking change. I do think we have a lot of standardization efforts coming up around font size and form size compatibility and this feels like a step in that direction.